src/main/java/br/com/finsavior/model/dto/Name.java

```diff
@@ -0,0 +1,10 @@
+package br.com.finsavior.model.dto;
+
+import lombok.Data;
+
+@Data
+public class Name {
```

Comment:
Como não é Entidade vale deixar o nome como NameDTO
```suggestion
public class NameDTO {
```